packages/core/src/external/aws/s3.ts

```diff
@@ -110,9 +116,14 @@ export class S3Utils {
         .promise();
       return {
         exists: true,
+
         size: head.ContentLength ?? 0,
+
         contentType: head.ContentType ?? "",
+
         eTag: head.ETag ?? "",
+
+        dateCreated: head.LastModified,
```

Comment:
nit: Why all these new lines?

Reply:
result of a resolution of a merge conflict - will fix this